### DISCUSSION TITLE
Clear data

### DISCUSSION THREAD

**Discussion by Silther** [astubenbord/paperless-mobile#330] [Q&A]
Is it still necessary to clear the app data if a new update arrives?

**Comment by astubenbord** [comment]
I would recommend you to only clear data/perform fresh installation if something seems to be working incorrectly. I'll try to stick to major versions being marked appropriately (i.e. from 3.1.8 to 3.2.0 for instance) if there are breaking changes. For minor updates you can expect that no reinstall or clearing data will be necessary. But I cannot fully guarantee it because I cannot test every feature on all devices.
